modules/phase_field/include/vectorpostprocessors/GrainTextureVectorPostprocessor.h

```diff
@@ -0,0 +1,42 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#ifndef GRAINTEXTUREVECTORPOSTPROCESSOR
+#define GRAINTEXTUREVECTORPOSTPROCESSOR
+
+#include "ElementVectorPostprocessor.h"
+#include "EulerAngleProvider.h"
```

Comment:
for this class you could move the include into the .C file and use a forward declaration instead

```
class EulerAngleProvider;
```